Sdp/TimeDescription.cs

```diff
@@ -259,6 +259,11 @@ public override bool Equals(object obj)
             return base.Equals(obj);
         }
 
+        public override int GetHashCode()
+        {
+            return HashCode.Combine(this);
```

Comment:
Perhaps use NtpStartDateTime and NtpStopDateTime, this logic is essentially the same as not overriding from what I understand.